content/library/advanced-features/secrets-management.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Secrets management
+slug: /library/advanced-features/secrets-management
+---
+
+# Secrets management
+
+It is bad practice to store unencrypted secrets in a git repository. Suppose your application needs access to sensitive credentials. In that case, the recommended solution is to store those credentials in a file not committed to the repository and pass them as environment variables.
+
+Secrets management allows you to securely store and access secrets in your Streamlit app as environment variables.
```

Comment:
I put some suggested edits for this section in Notion

Reply:
Thank you so much! Fixed on 03560c3